frontend/models/feedback-task-model/record-field/RecordField.model.js

```diff
@@ -0,0 +1,16 @@
+import { Model } from "@vuex-orm/core";
+
+class RecordField extends Model {
+  static entity = "recordFields";
+
+  static fields() {
+    return {
+      id: this.uid(),
+      title: this.string(null).nullable(),
+      text: this.string(null).nullable(),
```

Comment:
I can imagine this is the name of the field, right? Maybe `name` is more specific than `text`. Please, correct me if I'm wrong in any case.

Reply:
I am not sure it's better in the point of view of front, since name need to keep generic for the "stupid" component. 
More specifically, this recordField entity is used in the RecordFeedback.component => this component is only to render bloc of title with a specific text. So to not have business naming in stupid component, I need to convert some of the naming in generic naming